SeQuant/core/tensor.hpp

```diff
@@ -164,14 +164,22 @@ class Tensor : public Expr, public AbstractTensor, public Labeled {
     return bra_rank();
   }
 
-  /// @brief Convert Fock and Coulomb integrals to std::wstring formula
-  /// compatible with MPQC's integral factory DSL
+  /// @brief Convert Fock, Coulomb and perturbation integrals to std::wstring
+  /// formula compatible with MPQC's integral factory DSL
   /// @param df return density-fitted or regular integals
   /// @return formula in wstring format
-  /// @pre `rank() <= 2 && (label()=="f" || label()=="g")`
+  /// @note perturbation integrals are represented by μ operator, the type of
+  /// perturbation will be determined and corresponding integral will be
+  /// computed in MPQC
+  /// @pre `rank() <= 2 && (label()=="f" || label()=="g" || label()=="μ")`
   std::wstring to_mpqc_formula(bool df = false) const {
```

Comment:
this belongs to MPQC ... (it might already exist there...)

Reply:
It is not part of MPQC now. Where should it be moved to? Maybe `mpqc::formula` or `mpqc::detail` namespace?

Reply:
looks awfully similar to https://github.com/ValeevGroup/mpqc4/blob/4c5d48385fa64e54ff495ce246f0d1f5805a0991/src/mpqc/chemistry/qc/lcao/cc/cck.ipp#L875 ... no?

Reply:
think of how any sequant-using code in MPQC could avoid doing this? It can't, this is a basic building block.

Reply:
I am sure @cmasteran1 's code in MPQC would also do something like this.

Reply:
in ```
src/mpqc/chemistryqc/lcao/expression/sequant.cpp
```
there is a function:
```OrbitalIndex make_orbital_index(sequant::Index idx,
                                std::wstring unique_label = L"")
                                ```
which takes a sequant index and and converts to an mpqc orbital index. It uses a map to find the appropriate index to use.

Reply:
Addressed in https://github.com/ValeevGroup/mpqc4/pull/433

Reply:
> Addressed in [ValeevGroup/mpqc4#433](https://github.com/ValeevGroup/mpqc4/pull/433)

so it can be removed from here?

Reply:
Removed in https://github.com/ValeevGroup/SeQuant/pull/141/commits/07676024d9422a7d489ac64e0d28865581271a36